conf/logs.json

```diff
@@ -580,7 +584,16 @@
         "type": "string",
         "watchlist_id": "integer",
         "watchlist_name": "string"
-      }
+      },
+      "optional_top_level_keys": [
+        "alliance_data_srsthreat",
```

Comment:
id add all of the other alliance_ keys that exist in other ones too - i imagine theyll pop up

Reply:
good point @mime-frame I've added all options